lib/tx_build.ex

```diff
@@ -26,7 +26,7 @@ defmodule Stellar.TxBuild do
   def add_memo(tx, memo), do: impl().add_memo(tx, memo)
 
   @impl true
-  def set_time_bounds(tx, time_bounds), do: impl().set_time_bounds(tx, time_bounds)
```

Comment:
@CristhianRodriguezMolina @FelipeGuzmanSierra IMO, we shouldn't change the `TxBuild` API definition. My suggestion is to keep the `set_time_bounds/2` function and internally set the time bounds into the preconditions. We don't want to affect current SDK users

Reply:
@juanhurtado10  Already done, we added the function with the possibility to update the preconditions for every type `:none`, `:precond_time` and `:precond_v2`